external/vcm/vcm/types.py

```diff
@@ -0,0 +1,5 @@
+from typing import TypeVar
+import numpy as np
+import xarray as xr
+
+Array = TypeVar("Array", np.ndarray, xr.DataArray, float)
```

Comment:
If I let `Array` be anything i.e. `Array = TypeVar("Array")` then I get mypy errors when doing arithmetic such as `arr: Array * x: float`.

However if we don't include `tf.Tensor` here (since trying to avoid VCM dep on tf) I imagine this will cause issues for fv3fit type checking...

Reply:
Okay based on discussion with @nbren12 I'm deleting this type.